mix.exs

```diff
@@ -27,14 +27,14 @@ defmodule OIDC.MixProject do
 
   defp deps do
     [
-      {:dialyxir, "~> 1.0.0-rc.4", only: [:dev], runtime: false},
-      {:ex_doc, "~> 0.19", only: :dev, runtime: false},
-      {:hackney, "~> 1.0", optional: true},
-      {:jason, "~> 1.1"},
-      {:jwks_uri_updater, "~> 1.0"},
-      {:oauth2_metadata_updater, "~> 1.0"},
+      {:dialyxir, "~> 1.1", only: :dev, runtime: false},
```

Comment:
Forcing higher versions of these libraries seems unnecessary to me. Can you explain?